typescript-timers/interval.ts

```diff
@@ -5,18 +5,12 @@ if (!$timer)
 let count = 4;
 const intervalId = setInterval(function () {
   count--;
-  switch (count) {
-    case 3:
-      $timer.textContent = '3';
-      break;
-    case 2:
-      $timer.textContent = '2';
-      break;
-    case 1:
-      $timer.textContent = '1';
-      break;
-    case 0:
-      $timer.textContent = '~Earth Beeeelooowww Us~';
-      clearInterval(intervalId);
+  const countDown = count.toString();
+
+  if (count < 1) {
+    $timer.textContent = '~Earth Beeeelooowww Us~';
+    clearInterval(intervalId);
+  } else {
+    $timer.textContent = countDown;
```

Comment:
This works but personally I would look at this as `if (count > 0)` rather than `if (count < 1)`. This is because most of are values are going to be greater than zero whereas we only have it going below 1, one time.